jenkins/ci.suse.de/cloud-update-ci.yaml

```diff
@@ -0,0 +1,46 @@
+- job:
+    name: 'cloud-update-ci'
+    node: cloud-trigger
+
+    parameters:
+      - choice:
+          name: instance
+          choices:
+            - ci.suse.de
+            - ci.opensuse.org
+          description: |
+              ci.suse.de: deploys all CI jobs on ci.suse.de
+              ci.opensuse.org: deploys all CI jobs on ci.opensuse.org
+
+    logrotate:
+      numToKeep: 300
+      daysToKeep: -1
+
+    builders:
+      - shell: |
+          ## THIS JOB ONLY UPDATES JENKINS JOBS - NO WORKER CODE IN HERE - DO NOT DARE TO ADD SOME
+          ## THUS RUNNING ON THE DEDICATED TRIGGER NODE
+          [[ $NODE_NAME != 'cloud-trigger' ]] && exit 99
+          ##
+
+          set -x
+
+          export automationrepo=~/github.com/SUSE-Cloud/automation
+
+          # automation bootstrapping
+          if ! [ -e ${automationrepo}/scripts/jenkins/update_automation ] ; then
+            rm -rf ${automationrepo}
+            curl https://raw.githubusercontent.com/SUSE-Cloud/automation/master/scripts/jenkins/update_automation | bash
+          fi
+          # fetch the latest automation updates
+          ${automationrepo}/scripts/jenkins/update_automation
+
+          mtarget=test
+          case $instance in
+              ci.suse.de) mtarget=cisd_deploy ;;
+              ci.opensuse.org) mtarget=cioo_deploy ;;
+          esac
+
+          cd ${automationrepo}
+          make jjb_test
+          make $mtarget
```

Comment:
If `make jjb_test` fails, then jenkins will stop the job immediately. So no need for `&&`.